ui/app/AppLayouts/Wallet/SendModal.qml

```diff
@@ -6,37 +6,49 @@ import "../../../imports"
 import "../../../shared"
 import "./components"
 
-Item {
-    function open() {
-        popup.open()
-        sendModalContent.valueInput.text = ""
-        sendModalContent.valueInput.forceActiveFocus(Qt.MouseFocusReason)
+ModalPopup {
+    id: popup
+
+    title: qsTr("Send")
+
+    onOpened: {
+        sendModalContent.amountInput.text = ""
+        sendModalContent.amountInput.forceActiveFocus(Qt.MouseFocusReason)
         sendModalContent.defaultAccount = walletModel.getDefaultAccount()
+        const accounts = walletModel.accounts
+        const numAccounts = accounts.rowCount()
+        const accountsData = []
+        for (let i = 0; i < numAccounts; i++) {
+            accountsData.push({
+                name: accounts.rowData(i, 'name'),
+                address: accounts.rowData(i, 'address'),
+                iconColor: accounts.rowData(i, 'iconColor')
+            })
+        }
+        sendModalContent.accounts = accountsData
     }
 
-    function close() {
-        popup.close()
+    SendModalContent {
+        id: sendModalContent
+        accounts: []
     }
 
-    Popup {
-        id: popup
-        modal: true
-        closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
-        Overlay.modal: Rectangle {
-            color: "#60000000"
-        }
-        parent: Overlay.overlay
-        x: Math.round((parent.width - width) / 2)
-        y: Math.round((parent.height - height) / 2)
-        width: 480
-        height: 510
-        background: Rectangle {
-            color: Theme.white
-            radius: Theme.radius
-        }
-        padding: 0
-        contentItem: SendModalContent {
-            id: sendModalContent
+    footer: StyledButton {
+        anchors.top: parent.top
+        anchors.right: parent.right
+        anchors.rightMargin: Theme.padding
+        label: qsTr("Send")
+
+        onClicked: {
+            if (!sendModalContent.validate()) {
+                return;
+            }
+
+            let result = walletModel.onSendTransaction(sendModalContent.selectedAccountAddress,
+                                                       sendModalContent.toText,
+                                                       sendModalContent.amountText,
+                                                       sendModalContent.passwordText)
+            console.log(result)
```

Comment:
this `console.log` is here because there is no verification yet if the sendTransaction works or not, right?

Reply:
Yes, there is no result or whatever yet so we just log it for now